ci/image/build.sh

```diff
@@ -7,22 +8,17 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $THIS_DIR/../_init.sh
 
 cp -p $THIS_DIR/../../pom.xml $THIS_DIR
-cp -rp $THIS_DIR/../../dependencies/ $THIS_DIR
-
-for name in "${!BUILD_IMAGE_NAMES[@]}"; do
-    docker build \
-        --pull \
-        --file $THIS_DIR/Dockerfile.$name-build \
-        --label snowflake \
-        --label $DRIVER_NAME \
-        --tag ${BUILD_IMAGE_NAMES[$name]} .
-done
+mkdir -p dependencies && cp -rp $THIS_DIR/../../dependencies/ $THIS_DIR/dependencies
 
 for name in "${!TEST_IMAGE_NAMES[@]}"; do
+    echo "Building $name"
```

Comment:
why remove build and test as separate loop?

Reply:
We didn't use `build` images. This was an empty array. If we need it I can revert it, but it doesn't seem to be used at all.